scaper/core.py

```diff
@@ -1944,41 +2082,24 @@ def _generate_audio(self, audio_path, ann, reverb=None,
                             "audio of the isolated events will not add up to the "
                             "mixture", ScaperWarning)
 
-            # Finally combine all the files and optionally apply reverb.
-            # If there are no events, throw a warning.
-            if len(event_audio_list) == 0:
-                warnings.warn(
-                    "No events to synthesize (silent soundscape), no audio "
-                    "generated.", ScaperWarning)
-            else:                        
-                tfm = sox.Transformer()
-                if reverb is not None:
-                    tfm.reverb(reverberance=reverb * 100)
-                # TODO: do we want to normalize the final output?
-                soundscape_audio = sum(event_audio_list)
-                soundscape_audio = tfm.build_array(
-                    input_array=soundscape_audio,
-                    sample_rate_in=self.sr,
-                )
-                soundscape_audio = soundscape_audio.reshape(-1, self.n_channels)
-
-                # Save to disk if output path provided
-                if audio_path is not None:
-                    soundfile.write(audio_path, soundscape_audio, self.sr, subtype='PCM_32')
-
         # Document output paths
+        # TODO: this is redundant with data stored in ann.sandbox.scaper.generate,
```

Comment:
Don't think this matters too much but the comment says things are stored in `ann.sandbox.scaper.generate` when I think we store everything in `ann.sandbox.scaper`?

Reply:
we're not storing anything in `scaper.generate` (that was an earlier idea), but the duplicity still exists (for compatibility, for now). I've updated the comment to reflect this.